testapi.pm

```diff
@@ -390,8 +390,9 @@ sub upload_logs {
     my ($file) = @_;
 
     bmwqemu::log_call('upload_logs', file => $file);
-    my $cmd      = "curl --form upload=\@$file ";
     my $basename = basename($file);
+    my $upname   = ref($autotest::current_test) . '-' . $basename;
```

Comment:
We should fail if module names are duplicated in one test - the job module index in the DB is the name not the fullname. And we only show the name in the overview page. So let's not make this more complicated *here*